v2/modules/sync/index.js

```diff
@@ -51,60 +54,51 @@ const getSoC = (akey, callback) => {
-                            ], (err, dbRes) => {
-                                if (!err && dbRes) {
-                                    db.query('INSERT INTO statistics (akey, type, value, timestamp) VALUES (?, ?, ?, ?)', [
-                                        akey, 'normal_charge_port', extendedObj.normalChargePort, now
-                                    ], (err, dbRes) => {
-                                        if (!err && dbRes) {
-                                            db.query('INSERT INTO statistics (akey, type, value, timestamp) VALUES (?, ?, ?, ?)', [
-                                                akey, 'aux_battery_voltage', extendedObj.auxBatteryVoltage, now
-                                            ], (err, dbRes) => {
-                                                if (!err && dbRes) {
-                                                    db.query('INSERT INTO statistics (akey, type, value, timestamp) VALUES (?, ?, ?, ?)', [
-                                                        akey, 'dc_battery_voltage', extendedObj.dcBatteryVoltage, now
-                                                    ], (err, dbRes) => {
-                                                        if (!err && dbRes) {
-                                                            db.query('INSERT INTO statistics (akey, type, value, timestamp) VALUES (?, ?, ?, ?)', [
-                                                                akey, 'dc_battery_current', extendedObj.dcBatteryCurrent, now
-                                                            ], (err, dbRes) => {
-                                                                if (!err && dbRes) {
-                                                                    db.query('INSERT INTO statistics (akey, type, value, timestamp) VALUES (?, ?, ?, ?)', [
-                                                                        akey, 'dc_battery_power', extendedObj.dcBatteryPower, now
-                                                                    ], (err, dbRes) => callback(err, (!err && dbRes)));
-                                                                } else callback(err);
-                                                            });
-                                                        } else callback(err);
-                                                    });
-                                                } else callback(err);
-                                            });
-                                        } else callback(err);
-                                    });
-                                } else callback(err);
-                            });
-                        } else callback(err);
-                    });
-                } else callback(err);
-            });
-        } else callback(err);
-    });
+    try {
+        await promiseDbQuery(
```

Comment:
Maybe you can make a iterable function (e.g. create an array) to just change / modify the key / value